### PR TITLE
Feature: Midaz 415

### PR DESCRIPTION
**Pull request by maxwelbm**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Auth
- [ ] Infra
- [x] Ledger
- [ ] Mdz
- [ ] Transaction
- [ ] Audit
- [ ] Pipeline
- [x] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [x] I have updated the documentation accordingly.
- [ ] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please, always remember to target your PR to develop branch instead of main.

### CODE REVIEW

**Review by ClaraTersi — APPROVED** [review]

File: pkg/net/http/httputils.go [hunk 1]
```diff
@@ -27,6 +28,7 @@ type QueryHeader struct {
 	SortOrder    string
 	StartDate    time.Time
 	EndDate      time.Time
+	Alias        string
```
Comment (MartinezAvellan) [inline pkg/net/http/httputils.go:31]: Validate if alias should be +1

```suggestion
	Aliases        []string
```

File: pkg/mpostgres/pagination.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ type Pagination struct {
 	SortOrder  string    `json:"-" example:"asc"`
 	StartDate  time.Time `json:"-" example:"2021-01-01"`
 	EndDate    time.Time `json:"-" example:"2021-12-31"`
+	Alias      string    `json:"-" example:"@wallet_12345123"`
```
Comment (MartinezAvellan) [inline pkg/mpostgres/pagination.go:18]: Validate if alias should be +1

```suggestion
	Aliases        []string
```

File: pkg/net/http/httputils.go [hunk 3]
```diff
@@ -40,26 +42,24 @@ type Pagination struct {
...
-	sortOrder := "desc"
-	useMetadata := false
+	var (
+		metadata     *bson.M
+		portfolioID  string
+		toAssetCodes []string
+		startDate    time.Time
+		endDate      time.Time
+		cursor       string
+		alias        string
```
Comment (MartinezAvellan) [inline pkg/net/http/httputils.go:57]: Validate if alias should be +1

```suggestion
	Aliases        []string
```

File: pkg/net/http/httputils.go [hunk 4]
```diff
@@ -110,6 +112,7 @@ func ValidateParameters(params map[string]string) (*QueryHeader, error) {
 		SortOrder:    sortOrder,
 		StartDate:    startDate,
 		EndDate:      endDate,
+		Alias:        alias,
```
Comment (MartinezAvellan) [inline pkg/net/http/httputils.go:115]: Validate if alias should be +1

```suggestion
	Aliases:        aliases,
```

Comment (MartinezAvellan) [inline pkg/net/http/httputils.go:115]: Validate if alias should be +1
```suggestion
		Aliases:        aliases,
```

File: pkg/net/http/httputils.go [hunk 5]
```diff
@@ -273,6 +276,7 @@ func (qh *QueryHeader) ToOffsetPagination() Pagination {
 		SortOrder: qh.SortOrder,
 		StartDate: qh.StartDate,
 		EndDate:   qh.EndDate,
+		Alias:     qh.Alias,
```
Comment (MartinezAvellan) [inline pkg/net/http/httputils.go:279]: Validate if alias should be +1
```suggestion
		Aliases:     qh.Aliases,
```

**Review by MartinezAvellan — APPROVED** [review]

**Pull request merged**